src/style.css

```
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --header-hover-color: #00acdc;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.container {
+  max-width: 1200px;
+  margin: auto;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: inline-flex;
+}
+
+.header-body {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.menu-list {
+  display: flex;
+  align-items: center;
+}
+
+.menu-item {
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.menu-item:last-child {
+  margin-right: 0;
+}
+
+.menu-item:hover {
+  color: var(--header-hover-color);
+}
+
+.menu-link {
+  padding: 23px 0;
+}
+
+.is-active {
+  position: relative;
+  color: var(--header-hover-color);
+}
+
+.is-active::after {
+  content: " ";
+  position: absolute;
+  border-bottom: 4px solid var(--header-hover-color);
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

pseudo-element ::after is not a border. Just make it as an element
```suggestion
.is-active::after {
  content: "";
  position: absolute;
  width: 100%;
  height: 4px;
  background-color: var(--header-hover-color);
  bottom: 0;
  left: 0;
}
```